CONTRIBUTING.md

```diff
@@ -296,6 +299,75 @@ follow the Carbon documentation and coding styles.
     -   Keep in mind that code contribution guidelines are incomplete while we
         start work on Carbon, and may change later.
 
+#### How to say things
+
+-   Treat others with respect
+-   Recognize that other points of view are valid, and most decisions are about
+    choosing the best set of trade-offs or closest alignment with Carbon's goals
+    since there usually isn't a single best answer. It is important to frame
+    feedback and discussion about someone else's proposal under the assumption
+    that they too have deep experience in the area but may have come to a
+    different conclusion.
+-   Be clear when something is an opinion by using "I" or "me":
+    -   Not as helpful: `foo` is objectively better word to use
+    -   More helpful: I find `bar` confusing since it has this alternate
+        meaning, I think `foo` is clearer.
+-   It can be helpful to define your terms.
+
+When trying to make a point, please employ these strategies to make your
+argument _effective_ and _helpful_:
+
+-   Focus should be on explaining the basis by which others can come to a
+    conclusion. Generally this means connecting a potential solution to the use
+    cases it helps with.
+-   Be specific and concrete, using examples to demonstrate the benefits and
+    disadvantages of the different options
+    -   Minimally helpful: "I like &lt;X>", "Carbon should have feature &lt;Y>"
+    -   More helpful: "I think Carbon should have feature &lt;Z>, it would mean
+        &lt;this example> would be written &lt;like this> instead of &lt;like
+        that>." (Assuming the reader will think the example is representative of
+        an important class of use cases.)
+    -   Very helpful: "If we go with approach &lt;X>, you can see it helps with
+        problem &lt;P> in &lt;this example> of use case &lt;A>. It doesn't help
+        with use case &lt;B>, but that can be better solved by &lt;Y>, as can be
+        seen in &lt;other example>."
+    -   Very helpful: "Yes &lt;X> gives a nice answer in that case, but it has a
+        problem / I don't see how it applies in &lt;this other situation>."
+-   Explain the reasoning by which you can come to your conclusion
+    -   Avoid [fallacies](https://en.wikipedia.org/wiki/List_of_fallacies) like
+        [arguing from authority](https://en.wikipedia.org/wiki/Argument_from_authority)
+    -   Don't expect people to read long works to understand your point
+
+If someone questions or argues with your point, try to directly address the
+points being made. Avoid evading by stepping backwards or switching to a more
+general or more meta level.
```

Comment:
I found this ambiguous: "(Avoid evading) by stepping backwards" versus "Avoid (evading by stepping backwards)" -- are we encouraging or discouraging taking a step back?

```suggestion
points being made. Try not to step backwards or switch to a more general
or more meta level as that can seem like you're evading the question.
```